utils/oc.py

```diff
@@ -15,12 +17,25 @@ class JSONParsingError(Exception):
 
 
 class OC(object):
-    def __init__(self, server, token):
-        self.oc_base_cmd = ['oc', '--server', server, '--token', token]
+    def __init__(self, server, token, jh=None):
+        oc_base_cmd = ['oc', '--server', server, '--token', token]
+
+        if jh is not None:
+            jump_host = JumpHost(jh)
+            oc_base_cmd = jump_host.ssh_base_cmd + oc_base_cmd
+            self.jump_host = jump_host
+
+        self.oc_base_cmd = oc_base_cmd
 
     def whoami(self):
         return self._run(['whoami'])
 
+    def cleanup(self):
+        if not hasattr(self, 'jump_host'):
```

Comment:
`if not hasattr(self, 'jump_host') and isinstance(self.jump_host, JumpHost)`

Reply:
(what I want to do is to prevent that for some reason jump_host is None or something)

Reply:
fixed (`or not` instead of `and`)